packages/wrangler/src/config/environment.ts

```diff
@@ -450,6 +462,17 @@ export interface EnvironmentNonInheritable {
 	 */
 	cloudchamber: CloudchamberConfig;
 
+	/**
+	 * Container app configuration
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default `{}`
+	 * @nonInheritable
+	 */
+	container_app: ContainerApp[];
```

Comment:
Bikeshedding on the name, but do you have a link to any internal discussion that led to the `container_app` name?

Reply:
Hey @penalosa, I don't think we've got a link, but this was something that we landed on for the public name of these apps.

I do think this would be annoying to change after the fact though, so I'll kick off a convo with our team just to make sure we're all feeling good here and nobody has a last second veto.

"Application" is pretty set in stone for us on Cloudchamber at least, and then we were adding "container" here to differentiate it from Workers apps. IMO it's the right name for now, but will make sure we've got near-consensus.

Reply:
We talked about this a bunch as a team, and "[[containers.app]]" is what we want to go with. - App is approachable and we needed a clear name for group of container instances.